src/api/user/auth.js

```diff
@@ -70,3 +70,38 @@ exports.confirm = wrapper(async (req, res, next) => {
   await user.update({ isVerified: true })
   res.json('Success confirmed')
 })
+
+// TODO need to destroy previous JWT Token ------------------
+exports.changePassword = wrapper(async (req, res, next) => {
+  const result = await jwtPromiseVer(req.body.token, secret)
+  const user = await User.findOne({ where: { email: result.email } })
+  if (!user) return next({ httpCode: 404, message: 'No such user found' })
+
+  const successCompare = await bcrypt.compare(req.body.password, user.password)
+  if (!successCompare) return next({ httpCode: 401, message: 'Password doesn’t match' })
+
+  const hashedPassword = await bcrypt.hash(req.body.password_new, saltRounds)
+  await user.update({ password: hashedPassword })
+  res.json('Successfully changed!')
+})
+
+// TODO need to destroy previous JWT Token №2 ------------------
+exports.resetPassword = wrapper(async (req, res, next) => {
+  const result = await jwtPromiseVer(req.query.token, 'forgot_password')
+  const user = await User.findOne({ where: { email: result.email } })
+  if (!user) return next({ httpCode: 404, message: 'No such user found' })
```

Comment:
Actually unnecessary check because you wouldn't create token with email that doesn't exist